README.md

```diff
@@ -181,6 +181,17 @@ Then, specify all the required certificates that need to be completed before usi
 ```
 Default is an empty list.
 
+### File mapping configuration
+Windmill lists unmapped files by checking Indexd. Before centralized auth's deployment, a file was determined to be
+unmapped by checking if the acl field was null. After centralized auth, we use the authz field instead. The manifest
+configuration setting `use_indexd_authz` specifies which of the two fields should be used. This field defaults to false.
+```
+"global": {
+    ...
+    "use_indexd_authz": "true"
```

Comment:
```suggestion
    "use_indexd_authz": true
```

Reply:
hmm are you sure about this one? i see quotation marks around boolean values in production manifests

Reply:
yeah in manifest.json there are `"Ture"`, `"true"` and `true`...

in portal configs we use `true` only. I think any of these styles will work, just trying to make a bit of consistance. But no big deal if leave as is

Reply:
Ah, but this is not a portal config setting, it is a manifest.json setting :O